ledger/participant-integration-api/src/main/scala/platform/store/FlywayMigrations.scala

```diff
@@ -32,14 +36,16 @@ private[platform] class FlywayMigrations(jdbcUrl: String)(implicit loggingContex
     }
 
   def migrate(
-      allowExistingSchema: Boolean = false
+      allowExistingSchema: Boolean = false,
+      enableExperimentalSchema: Boolean = false,
```

Comment:
rename to appendOnly (e.g. enableAppendOnlySchema): this will be the same flag probably throughout the whole rollout